source/docs/magento2/search-engine.md

```diff
@@ -0,0 +1,72 @@
+---
+id: m2-search-engine
+title: Search engine
+---
+
+When configured with Magento2, Front-Commerce can leverage Elasticsearch to bring a search engine to your website.
+
+## Elasticsearch
+
+### Requirements
+
+On Magento2 side, Front-Commerce requires [Elasticsuite](https://github.com/Smile-SA/elasticsuite) and [its CMS Page search plugin](https://github.com/Smile-SA/magento2-module-elasticsuite-cms-search) to be installed.
+
+The Elasticsuite installation and configuration procedure can be found [in a dedicated page on the project wiki](https://github.com/Smile-SA/elasticsuite/wiki/ModuleInstall) while for the CMS Page search plugin, [the setup is detailed in its readme document](https://github.com/Smile-SA/magento2-module-elasticsuite-cms-search#how-to-use).
+
+### Front-Commerce configuration
+
+First, you need to make sure the Elasticsearch search client is installed with a version that matches your ElasticSearch server version (6 or 7):
+
+```
+npm i @elastic/elasticsearch@6
```

Comment:
I suggest we document the most recent one (more likely the version used for new projects)
```suggestion
npm i @elastic/elasticsearch@7
```